xml/http_servers.xml

```diff
@@ -2,6 +2,21 @@
 <fingerprints matches="http_header.server" protocol="http" database_type="service" preference="0.90">
   <!-- HTTP Server headers are matched against these patterns to fingerprint HTTP servers. -->
 
+	<fingerprint pattern="^BASHttpd/([\d.]+)">
+    <description>BASHttpd</description>
+    <example service.version="4.3.24">BASHttpd/4.3.24-release</example>
+    <param pos="0" name="service.product" value="bashttpd"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+
+  <fingerprint pattern="^monit ([\d.]+)$">
+    <description>Monit</description>
+    <example service.version="5.6">monit 5.6</example>
+    <param pos="0" name="service.vendor" value="Tildeslash"/>
+    <param pos="0" name="service.product" value="monit"/>
```

Comment:
Where possible please use the same casing that the vendor uses in names. In this case it would be `Monit`.